synapse/rest/client/room.py

```diff
@@ -1113,6 +1114,15 @@ async def _do(
     ) -> Tuple[int, JsonDict]:
         content = parse_json_object_from_request(request)
 
+        # Ensure the redacts property in the content matches the one provided in
+        # the URL.
+        room_version = await self._store.get_room_version(room_id)
+        if room_version.msc2176_redaction_rules:
+            if "redacts" in content and content["redacts"] != event_id:
+                raise SynapseError(400, "Cannot provide", Codes.INVALID_PARAM)
```

Comment:
```suggestion
                raise SynapseError(400, "Cannot provide a redacts value incoherent with the event_id of the URL parameter", Codes.INVALID_PARAM)
```
`Cannot provide` seems a bit cryptic as an error message.

Reply:
Oops! I guess I never finished writing this out. Thank you!